src/sections/header.liquid

```diff
@@ -121,10 +115,14 @@
         "label": "Logo image"
       },
       {
-        "type": "text",
+        "type": "range",
         "id": "logo_max_width",
+        "min": 50,
+        "max": 450,
+        "step": 8,
```

Comment:
No, it seems like there aren't any yet for headers. I understood how you got 8 but was just wondering if we had established a pattern for this. Bump it to 10? 😛